lib/logstash/dead_letter_post_office.rb

```diff
@@ -0,0 +1,55 @@
+# encoding: utf-8
+
+class LogStash::DeadLetterPostOffice
+
+  def self.logger=(logger)
+    @logger = logger
+  end
+
+  def self.logger
+    @logger ||= Cabin::Channel.get(LogStash)
+  end
+
+  def self.destination=(destination)
+    logger.info("Dead letter events will be sent to \"#{destination.location}\".")
+    @destination = destination
+  end
+
+  def self.<<(events)
+    events = [events] unless events.is_a?(Array)
+
+    events.each do |event|
```

Comment:
the << api feels weird to accept an array partly because `Array#<<` appends an object to an array, not concatonating an array to another array (that uses Array#+)